### ISSUE TITLE
Feature Request: Ability to make adjustments to other themes without having to swap to them

### ISSUE THREAD

**Issue by AlexGStapleton** [siteorigin/so-css#29]
[Reference thread](https://siteorigin.com/thread/how-to-use-siteorigin-css-editor-with-apppresser/)

Basically, the user is using a plugin that allows for a separate mobile theme. So, on desktop he is using theme A (with a child theme) and on mobile he's using Theme B (with a child theme). The user wants to know if it's possible (which is obviously isn't) to use SiteOrigin CSS with this setup.

Long story short, we should add the ability to change "theme" in SiteOrigin custom CSS without directly changing theme the WordPress theme. It's much the same as the standard WordPress plugin / theme editor. You can easily switch to another theme' "styling" from a drop down.

Reference Image:

WordPress Theme Editor Dropdown
![](https://i.imgur.com/OrOF0Km.png)

**Comment by gregpriday** [comment]
This would be a nice to have feature. Outside of what I'd consider an 80% feature, but something that a few users could find really useful.

**Comment by gregpriday** [comment]
Closing this issue for now. It's something we might address in the future, but not at the moment.
